Update README.md (#1)
churkin/eslint-config-devextreme: README.md

## TypeScript

- **Usage**    
    Add the following line to your *.eslintrc* file:
    
    ```javascript
    'extends': ['devextreme/typescript']
    ```

- **Required plugins**
  - [@typescript-eslint](https://github.com/typescript-eslint/typescript-eslint)
  
- **Extended plugins and configurations**
  - [`plugin:@typescript-eslint/recommended`](https://github.com/typescript-eslint/typescript-eslint/tree/master/packages/eslint-plugin#recommended-configs)
  - [`airbnb-typescript`](https://github.com/iamturns/eslint-config-airbnb-typescript)
  
## Spell Check

Lists words that the spell check should ignore.

- **Usage**
    Add the following line to your *.eslintrc* file:
    
    ```javascript
    "extends": ["devextreme/spell-check"]
    ```

- **Required plugins**
  - [spellcheck](https://github.com/aotaduy/eslint-plugin-spellcheck)

## Renovation Declarations

- **Usage**    
    Add the following line to your *.eslintrc* file:
    
    ```javascript
    'extends': ['devextreme/renovation-declarations']
    ```
    
- **Extended configurations**
  - [`devextreme/typescript`](#typescript)

- **Overrides**

>   - Allow importing devDependencies when in development mode:

Reviewer: ```suggestion
  - Allow devDependencies to be imported when in development mode:
```

    ```javascript
    'import/no-extraneous-dependencies': ['error', { devDependencies: true }]
    ```

  - Allow two classes (component class and props class) in one file:
  
    ```javascript
    'max-classes-per-file': ['error', 2]
    ```

>   - Do not require React. Declarations do not need it because they are not proper React components:

Reviewer: ```suggestion
  - Disable React dependency validation. Declarations do not require this dependency, since they are not proper React components. 
```

  
    ```javascript
    'react/react-in-jsx-scope': ['off']
    ```

## Jest Tests

- **Usage**    
    Add the following line to your *.eslintrc* file:
    
    ```javascript
    'extends': ['devextreme/jest']
    ```

- **Environment**
  - `node` (Node.js global variables and scope)
  - `jest` (Jest global variables)
  
- **Required plugins**
  - [jest](https://github.com/jest-community/eslint-plugin-jest)
  - [jest-formatting](https://github.com/dangreenisrael/eslint-plugin-jest-formatting)
  
- **Extended plugins**
  - [`plugin:jest/recommended`](https://github.com/jest-community/eslint-plugin-jest#recommended)
  - `plugin:jest-formatting/recommended`
  
- **Overrides**

>   - Allow using devDependencies in the tests:

Reviewer: Allow the use of devDependencies in tests:

please make this change to the other sentences below too
